hack/common

```diff
@@ -27,12 +28,12 @@ k8s_env() {
     if [[ "$__k8s_env" == "" ]]; then
         __k8s_env=$(kubectl get node -o jsonpath="{.items[*].metadata.name}")
```

Comment:
This command fails because kubectl is looking for the default kubeconfig which is not the one pointing to kind cluster. The problem is that at this point you don't know which environment are using and therefore must rely on either the default kubectl config file or the `KUBECONFIG` environment variable to be set correctly.

Reply:
The general assumption of all `hack` scripts is that `kubectl` already points to the right cluster. This has to be assured by the caller by either having the right `kubeconfig` at his home directory or `export`ing the correct `KUBECONFIG`.